packages/frontapp/src/app/components/BookCard/index.tsx

```diff
@@ -85,13 +94,47 @@ const BookCard: FC<IBookCardProps> = ({
   const handleItemRedirect = () => {
     navigate(`/item/${id}`);
   };
+  const [statusText, setStatusText] = useState('');
+  const currentStatus = getStatus(status?.status, date);
+  useEffect(() => {
+    console.log(currentStatus);
+    switch (currentStatus) {
+      case 'Free':
+        setStatusText('On the shelf');
+        break;
+      case 'Busy': {
+        const day = `${getDates(date).returnDate.getDate()}`.padStart(2, '0');
+        const month = `${getDates(date).returnDate.getMonth() + 1}`.padStart(
+          2,
+          '0'
+        );
+        setStatusText(`Return till: ${day}.${month}`);
+        break;
+      }
+      case 'Prolong': {
+        const day = `${getDates(date).returnDate.getDate()}`.padStart(2, '0');
+        const month = `${getDates(date).returnDate.getMonth() + 1}`.padStart(
+          2,
+          '0'
+        );
+        setStatusText(`Return till: ${day}.${month}`);
```

Comment:
That's not good approach. It's hard to apply localization with it and reuse formatting. We should use some sort of date formatting library, such as `luxon`, but you could suggest other options.

Reply:
i added hook that does everything for getting time, i think it replaces the need for luxon